client/src/modules/invoices/patientInvoice.service.js

```diff
@@ -135,6 +135,7 @@ function PatientInvoiceService(
     { key : 'reversed', label : 'FORM.INFO.CREDIT_NOTE' },
     { key : 'defaultPeriod', label : 'TABLE.COLUMNS.PERIOD', valueFilter : 'translate' },
     { key : 'debtor_group_uuid', label : 'FORM.LABELS.DEBTOR_GROUP' },
+    { key : 'project_id', label : 'FORM.LABELS.PROJECT' },
```

Comment:
Do we really need to put the project names here?  Shouldn't it be obvious from the identifiers? `PA.HBB.XXX`?

Reply:
It's true it's more information, but it can still help other users like the Administrator and the Director